test_runner/regress/test_compute_ctl.py

```diff
@@ -193,8 +193,9 @@ def test_sync_safekeepers_logs(neon_env_builder: NeonEnvBuilder, pg_bin: PgBin):
             timeout=10,
         )
     except TimeoutExpired as exc:
-        ctl_logs = exc.stderr.decode("utf-8")
-        log.info("compute_ctl output:\n" + ctl_logs)
+        if exc.stderr is not None:
```

Comment:
Shouldn't this be an `assert`? Output is probably always captured by `raw_cli`.

If not, I'd use `(exc.stderr or b"").decode(...)` to make sure log line is always there.